src/main/java/com/faforever/api/config/elide/checks/IsUser.java

```diff
@@ -0,0 +1,14 @@
+package com.faforever.api.config.elide.checks;
+
+import com.yahoo.elide.security.User;
+import com.yahoo.elide.security.checks.UserCheck;
+
+/**
+ * @author Dragonfire
+ */
+public class IsUser extends UserCheck {
```

Comment:
Would `IsAuthenticated` be more descriptive? Because there can also be anonymous users.